### DISCUSSION TITLE
[request] add /cmi data

### DISCUSSION THREAD

**Discussion by p82183645** [Zrips/CMI#1759] [Feature Requests]
please add about data command

/cmi data <number>
Change the data of the item
This is very useful, please add
Thank

**Cmi Version (using`/cmi version`):** 8.6.2.0

**Server Type (Spigot/Paperspigot/etc):** paper

**Server Version (using `/ver`):** 1.13.2

**Comment by Zrips** [comment]
Which data? NBT?

**Comment by p82183645** [comment]
> Which data? NBT?

damage data
like  sword:15

**Comment by p82183645** [comment]
> Which data? NBT?

can add? thank
